config/test_cases/test_branch_origin_extended_conformance_gce.yml

```diff
@@ -4,6 +4,12 @@ extensions:
   sync_repos:
     - name: "release"
   parameters:
+    - name: "SUITE"
```

Comment:
Yeah, this should not be in the `conformance` job ... but the tools today do not let you change that simply so ... whatever